### ISSUE TITLE
UI for Predict API

### ISSUE THREAD

**Issue by kwombach12** [kubecost/cost-analyzer-helm-chart#1992]
**What problem are you trying to solve?**
Create an experience in the Kubecost UI for predicting the cost of a deployment

**How would users interact with this feature?**
Upload deployment file and receive predict costs

**Comment by akramwaseem** [comment]
We can use the existing deployments for cost predicting as well. Which Kube cost already has data and predict the cost model for replicas more than what Kube cost shows already. 

We should be able to look at the cost predict for 24H and 1 day and 7 days and maybe couple of months as well.
Please feel free to ask questions if needed?

For new deployments(Never deployed before) we can upload a yaml.

**Comment by wam1punch** [comment]
Any update on this?
What version are we expecting this to be in?

**Comment by dwbrown2** [comment]
@wamak9 we just launched (101 release) the first audit UI to show recent cost events. Would love to get your input on this to see if you find it valuable. We're now exploring other UI additions as well.

**Comment by wamak9** [comment]
@dwbrown2  -- I did try to look at the Audit UI and I did enable it on most of the secondary clusters and primary cluster.
based on what I saw, I think Audit UI only works on Primary and doesn't do much or anything at all on gathering info from Secondary clusters. 

Is that expected Behavior ? 
If so, I am not sure, how that would be helpful with Predict UI ?

**Comment by dwbrown2** [comment]
@wamak9 that's correct, we've started implementing this feature as single cluster only with certain events. Any feedback on the core offering before we expand in these other areas?

**Comment by wamak9** [comment]
@dwbrown2  -- To be honest, I can only enable the audit on the Primary and with our setup we dont have anything else running on the same cluster. so there is not much events I can see or any chance for audit to gather.
So, I am not sure I am provide any feedback here at all.

**Comment by dwbrown2** [comment]
Ok, interesting. Just curious, any reason why you're not able to turn on for other clusters? That's what other users are doing. 

We're adding more features to single cluster now, but we very much to expect to bring this to multi-cluster!

**Comment by wamak9** [comment]
What I mean was, I did enable it on secondary clusters but I don’t see any events getting recorded.
So, either there are no changes which reports increase in cost or nothing is being collected.
@keithhand

**Comment by wamak9** [comment]
bump

**Comment by dwbrown2** [comment]
@wamak9 there are no known issues here, we'll be adding more events in the near future though. Are you saying there are deployment create and delete events not getting tracked? If so, would you be able to open a support ticket @keithhand?

**Comment by wamak9** [comment]
bump
